### ISSUE TITLE
Uncommitted migrations

### ISSUE THREAD

**Issue by hbradlow** [PetrDlouhy/django-admin-charts#55]
When running `python manage.py makemigrations --check --dry-run` we get the following uncommitted migrations:

```
Migrations for 'admin_tools_stats':
  /usr/local/lib/python3.10/site-packages/admin_tools_stats/migrations/0021_alter_cachedvalue_id_alter_criteriatostatsm2m_id_and_more.py
    - Alter field id on cachedvalue
    - Alter field id on criteriatostatsm2m
    - Alter field id on dashboardstats
    - Alter field id on dashboardstatscriteria
 ```

Is there a reason these are not included?

**Comment by PetrDlouhy** [comment]
Are you using BigInt as your default ID field?

**Comment by hbradlow** [comment]
Ah yes you are right `DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"`

Is there a way to change this setting for `django-admin-charts`?

**Comment by PetrDlouhy** [comment]
@hbradlow Probably best would be to set the `default_auto_field` for AppConfig in `django-admin-charts`. I will make PR for that.

**Comment by PetrDlouhy** [comment]
@alvarantson I have released version 1.3.0 where the BigAutoFields are app default and migrations should be updated. Please check.
